.eslintrc.js

```diff
@@ -5,15 +5,7 @@ module.exports = {
     project: './tsconfig.json',
     sourceType: 'module'
   },
-  overrides: [
-    {
-      files: [
-        '**/*.spec.ts',
-        '**/*.mock.ts'
-      ],
-      env: {
-        jest: true,
-      },
-    },
-  ],
-}
+  env: {
```

Comment:
Why was this change needed? Seems like it makes sense to only set the `jest` env if we are in spec/mock files?